docs/src/main/paradox/index.md

```diff
@@ -0,0 +1,140 @@
+# Lagom Akka gRPC Quickstart with Scala
+
+Lagom is an open source framework (built on Akka and Play) for developing reactive microservice systems in Java or Scala.
+Akka gRPC is a toolkit for building streaming gRPC servers and clients on top of Akka Streams.
+
+This Guide will show you ho to use Akka gRPC as an alternate RPC library to communicate two microservices developed using Lagom. 
+
+## Downloading the example.
+
+The Lagom Akka gRPC Quickstart is a [GitHub repository](https://github.com/lagom/akka-grpc-lagom-quickstart-scala) that you can clone locally:
```

Comment:
Should this also be available here: https://developer.lightbend.com/start/?group=lagom. Just like https://developer.lightbend.com/start/?group=akka&project=akka-grpc-quickstart-scala

Reply:
I'm not sure. Doesn't https://developer.lightbend.com/start/?group=lagom require the target projects (anything listed) to be a `.g8` or maven archetype?

In any case, I think we can postpone making the change until all the pieces are connected and this is already listed there.

Reply:
Then, yes! let's add an issue about it.

Reply:
https://github.com/lagom/akka-grpc-lagom-quickstart-scala/issues/16